src/lib.rs

```diff
@@ -1172,6 +1172,10 @@ pub unsafe extern "C" fn wgpuCommandEncoderBeginRenderPass(
             make_slice(descriptor.colorAttachments, descriptor.colorAttachmentCount)
                 .iter()
                 .map(|color_attachment| {
+                    if color_attachment.depthSlice != native::WGPU_DEPTH_SLICE_UNDEFINED {
+                        unimplemented!("Depth slice on color attachments is not implemented");
```

Comment:
We can't panic by default because it will break pretty much all existing code, a `log::warn` would be better here like I mentioned in previous review.